tests/unit/src/test/scala/tests/SemanticdbSuite.scala

```diff
@@ -35,3 +41,16 @@ class SemanticdbSuite extends DirectoryExpectSuite("semanticdb") {
     }
   }
 }
+
+class SemanticdbScala2Suite
+    extends SemanticdbSuite(InputProperties.scala2(), "semanticdb")
+
+/**
+ * There is a number of issues observed in Scala 3 that can be seen in the tests:
+ * - extension methods https://github.com/lampepfl/dotty/issues/11690
```

Comment:
Added the links here, otherwise the comments would get duplicated in the different expect suites